src/scicookie/{{cookiecutter.project_slug}}/build-system/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+{% if cookiecutter.build_system == "scikit-build-core" -%}
```

Comment:
where are you declaring this variable? `SKBUILD_PROJECT_NAME`

Reply:
it doesn't look correct to resuse the name sk... for both build system
also the first two lines seem to be duplicated .. probably you could move it to some first lines in order to reuse that for both build system